code_examples/sdk_examples/src/core_features/getting_started/05_fetch_endpoint_data.ts

```diff
@@ -1,13 +1,12 @@
 import axios from 'axios'
```

Comment:
@aybarsayan Not introduced by you, but this may be a good time to remove the axios package and replace it with a native fetch to make the HTTP requests.

Reply:
Yes, I can change this in a new pr 😊